### PR TITLE
Replaced the position absolutes with flex instead in Layers view

### PR DESCRIPTION
**Pull request by mapsmarketing**
Hi there,

I've only just recently come across this amazing software and felt I could contribute. My contribution is small and it mainly tackles the layers panel. I noticed that `position: absolute;` is used quite a lot to position the elements which I thought was unnecessary since it can be done more cleanly with `display: flex;` instead.

This is my first time contributing, so please let me know if I've done anything wrong or if further details are required.

Thank you very much

### CODE REVIEW

File: src/styles/scss/_gjs_layers.scss [hunk 1]
```diff
@@ -100,13 +113,10 @@ $layerNameSpacing: 5px !default;
     }
 
     &-name {
-      padding: $layerNameSpacing 0;
       display: inline-block;
       box-sizing: content-box;
-      overflow: hidden;
-      white-space: nowrap;
```
Comment (artf) [inline src/styles/scss/_gjs_layers.scss:107]: Any reason behind removing these? Those properties are used to apply ellipsis to long layer names.

**Review by artf — CHANGES_REQUESTED** [review]
Thank you @mapsmarketing, I really appreciate this PR ❤️

Besides a few comments, I'd also ask to check this regression with borders I see here
Screenshot 2023-10-08 at 16 26 06

File: src/styles/scss/_gjs_layers.scss (hunk 1 above)
Comment (mapsmarketing) [reply src/styles/scss/_gjs_layers.scss:107]: Thank you very much on your feedback @artf :)

It's dark-on-dark and I'm blind as a bat, so didn't see the border underneath. Will fix this up and the CSS code for the ellipsis. I'll resubmit once I've got it updated.

Comment (mapsmarketing) [reply src/styles/scss/_gjs_layers.scss]: Hi @artf 

I've made the requested fixes. Please let me know if you require anything further.

Thank you very much Artur

**Comment by artf** [comment]
@mapsmarketing I made some other small fixes/changes but overall you did a great job, thank you very much 🙇‍♂️

**Comment by mapsmarketing** [comment]
Glad it was useful and thank you very much @artf

**Pull request merged**